usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -107,6 +107,9 @@ def construct_contract_response(requested_award_dict):
     response["recipient"] = create_recipient_object(transaction)
     response["place_of_performance"] = create_place_of_performance_object(transaction)
 
+    parent_award = fetch_parent_award_from_piid(award["parent_award_piid"])
+    response["parent_generated_unique_award_id"] = parent_award["generated_unique_award_id"] if parent_award else None
```

Comment:
Preferably `fetch_parent_award_from_piid()` would return either the string value or None so this extra step outside the function isn't necessary.

```suggestion
    response["parent_generated_unique_award_id"] = fetch_parent_award_from_piid(award["parent_award_piid"])
```